python/paddle/fluid/dygraph/dygraph_to_static/utils.py

```diff
@@ -471,11 +480,15 @@ def remove_if_exit(filepath):
     else:
         module = SourceFileLoader(module_name, f.name).load_module()
     func_name = dyfunc.__name__
-    if not hasattr(module, func_name):
+    if hasattr(module, '__impl__'):
```

Comment:
and we can add some comment here to explain why special treatment

Reply:
Done,thx.